dev/breeze/src/airflow_breeze/breeze.py

```diff
@@ -23,24 +23,24 @@
     find_airflow_sources_root_to_operate_on,
 )
 
-from airflow_breeze.configure_rich_click import click  # isort: skip # noqa
+from airflow_breeze.configure_rich_click import click  # isort: skip  # noqa: F401
 
 find_airflow_sources_root_to_operate_on()
 create_directories_and_files()
 
 
-from airflow_breeze.commands import developer_commands  # noqa
-from airflow_breeze.commands.ci_commands import ci_group  # noqa
-from airflow_breeze.commands.ci_image_commands import ci_image  # noqa
-from airflow_breeze.commands.kubernetes_commands import kubernetes_group  # noqa
-from airflow_breeze.commands.production_image_commands import prod_image  # noqa
-from airflow_breeze.commands.release_management_commands import release_management  # noqa
-from airflow_breeze.commands.minor_release_command import create_minor_version_branch  # noqa
-from airflow_breeze.commands.release_command import airflow_release  # noqa
-from airflow_breeze.commands.release_candidate_command import release_management  # noqa
-from airflow_breeze.commands.sbom_commands import sbom  # noqa
-from airflow_breeze.commands.setup_commands import setup  # noqa
-from airflow_breeze.commands.testing_commands import group_for_testing  # noqa
+from airflow_breeze.commands import developer_commands  # noqa: I001, E402, F401
+from airflow_breeze.commands.ci_commands import ci_group  # noqa: E402
+from airflow_breeze.commands.ci_image_commands import ci_image  # noqa: E402
+from airflow_breeze.commands.kubernetes_commands import kubernetes_group  # noqa: E402
+from airflow_breeze.commands.production_image_commands import prod_image  # noqa: E402
+from airflow_breeze.commands.release_management_commands import release_management  # noqa: E402
+from airflow_breeze.commands.minor_release_command import create_minor_version_branch  # noqa: E402, F401
+from airflow_breeze.commands.release_command import airflow_release  # noqa: E402, F401
+from airflow_breeze.commands.release_candidate_command import release_management  # noqa: E402, F811
```

Comment:
**F811**: `release_management` imported two times from `airflow_breeze.commands.release_management_commands` and `airflow_breeze.commands.release_candidate_command`, I do not change because not sure may be first import required for something, because if it change it the command hashes/images regenerated